offscreen-canvas/the-offscreen-canvas/offscreencanvas.transfer.to.imagebitmap.w.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/canvas-tests.js"></script>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/scripting.html#the-offscreen-2d-rendering-context">
+
+<script id="myWorker" type="text/worker">
```

Comment:
Why not put this into a separate worker.js file? Then the test could be called ...worker.html